mls-rs/src/group/snapshot.rs

```diff
@@ -39,7 +41,8 @@ pub(crate) struct Snapshot {
     key_schedule: KeySchedule,
     #[cfg(feature = "by_ref_proposal")]
     pending_updates: SmallMap<HpkePublicKey, (HpkeSecretKey, Option<SignatureSecretKey>)>,
-    pending_commit: Option<CommitGeneration>,
+    #[mls_codec(with = "mls_rs_codec::byte_vec")]
+    pending_commit: Vec<u8>,
     signer: SignatureSecretKey,
```

Comment:
Doesn't this break a user who has already exported a snapshot? I think you need v2 and a custom MlsDecode to handle it?

Reply:
Note that the old CommitGeneration is not usable anymore, so the pending commit will be lost.